esque/controller/topic_controller.py

```diff
@@ -114,46 +111,12 @@ def get_cluster_topic(self, topic_name: str) -> Topic:
     def get_local_topic(self, topic_name: str) -> Topic:
         return Topic(topic_name)
 
-    def get_offsets_closest_to_timestamp(self, topic_name: str, timestamp_limit: pendulum) -> Dict[int, int]:
+    def get_offsets_closest_to_timestamp(
```

Comment:
This is basically the same method as offsets_for_timestamp in client.py. Maybe move that one here, and then this one is obsolete?